packages/render-fidelity-tools/test/config.json

```diff
@@ -1,4 +1,4 @@
-{
+{ 
```

Comment:
Small white-space changes like this usually indicate that clang-format hasn't been applied. If you resave this with clang-format enabled, I think this change will disappear.